utils/run_benchmarks.py

```diff
@@ -134,26 +139,35 @@ def run_benchmark(
     # First convert ground truth DD to csv
     if not skip_csv:
         shutil.rmtree(csv_folder, ignore_errors=True)
-        res = subprocess.run(
-            [
-                "python",
-                "utils/dd_to_csv.py",
-                dd_folder,
-                csv_folder,
-            ],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT,
-            text=True,
-        )
-        if res.returncode != 0:
-            # Remove partial outputs
-            shutil.rmtree(csv_folder, ignore_errors=True)
-            print(res.stdout)
-            print(f"ERROR: dd_to_csv failed on {benchmark['name']}")
-            sys.exit(1)
+        if os.name != "nt":
+            res = subprocess.run(
+                [
+                    "python",
+                    "utils/dd_to_csv.py",
+                    dd_folder,
+                    csv_folder,
+                ],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+                text=True,
+                # windows needs this for subprocess to inherit venv when calling python directly:
+                shell=True if os.name == "nt" else False,
```

Comment:
Doesn't run in virtualenv on windows without this.
I think the `xl2times` script subprocess call below must do its own venv activation, so it doesn't need this.

Reply:
Sounds good. But I'm not sure if we need this line, since this call is under a `os.name != nt` condition, so would never run on Windows?